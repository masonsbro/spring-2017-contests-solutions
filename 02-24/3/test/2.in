100
8 3 2 445
7 3 10 865
6 4 2 74
11 7 8 191
1 2 5 243
8 8 5 204
10 7 12 144
7 2 11 861
5 1 4 146
11 2 7 140
10 9 8 650
5 4 12 996
7 1 1 151
5 9 7 369
6 8 4 63
10 1 12 489
11 3 6 34
7 4 5 426
10 9 5 774
4 11 12 922
7 1 7 332
5 10 6 174
11 6 4 605
8 10 11 833
2 8 4 371
1 2 9 759
3 2 4 898
12 2 1 88
3 10 7 832
4 11 11 555
11 10 5 719
11 1 8 989
12 8 11 556
11 5 12 629
8 11 5 851
2 8 5 133
8 7 8 520
3 5 11 692
5 7 11 744
1 5 2 78
11 2 11 189
4 10 6 500
8 11 12 198
10 5 4 932
12 5 12 197
11 7 3 614
3 11 10 979
5 1 10 201
6 6 6 639
10 12 9 413
10 11 2 2
5 6 10 117
7 12 2 574
1 4 4 143
10 8 2 115
10 7 10 737
8 2 1 76
6 10 8 331
5 1 5 694
5 7 3 566
5 9 7 420
1 12 4 905
7 3 9 55
2 11 9 699
2 6 4 187
10 5 6 85
10 12 11 111
10 10 4 123
8 10 5 470
2 12 11 357
4 4 6 164
2 3 4 108
4 8 6 963
11 5 5 534
8 9 11 521
9 9 9 583
11 8 3 417
9 7 10 552
5 4 4 418
1 2 1 402
9 2 11 521
9 8 2 420
12 7 8 400
8 3 12 170
7 2 7 552
1 1 10 871
1 11 10 667
2 8 8 677
1 1 9 930
10 9 1 292
4 2 1 425
2 3 11 155
10 12 7 357
3 11 12 609
7 9 3 692
10 10 12 229
9 8 7 121
8 6 3 238
8 3 11 668
12 12 11 44
