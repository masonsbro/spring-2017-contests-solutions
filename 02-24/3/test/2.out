57159360
943853400
1598400
135539712
3557520
80294400
105235200
685776168
2575440
18239760
1828008000
1429937280
963984
258041700
4644864
172519200
1413720
152797680
1619595000
2695987008
32503464
54810000
580741920
3668132160
52996608
62298720
116251488
1127808
873250560
896116320
1708344000
516970080
1962208512
1569229200
1914137280
8554560
728232960
474760440
1280386800
369720
52141320
360720000
249651072
1043467200
168505920
523367460
1899651600
12180600
530012160
1107963360
7920
24850800
332690400
1976832
12806400
2284405200
561792
316488960
72349500
202180860
334189800
236139840
3492720
581288400
10124928
13158000
98461440
36604800
531288000
202444704
15586560
1695168
1069438464
471388500
1292363424
1489224528
276100704
1153867680
84068160
1944072
323090856
152772480
646732800
50232960
179490528
45570720
294066960
352516608
46754820
46200240
8690400
9575280
644142240
882660240
543816504
379224000
44640288
49146048
707876928
18817920
